composer.json

```diff
@@ -1,46 +1,62 @@
 {
-    "name":         "behat/mink-selenium2-driver",
-    "description":  "Selenium2 (WebDriver) driver for Mink framework",
-    "keywords":     ["selenium", "webdriver", "javascript", "ajax", "testing", "browser"],
-    "homepage":     "http://mink.behat.org/",
-    "type":         "mink-driver",
-    "license":      "MIT",
-
+    "name": "behat/mink-selenium2-driver",
+    "description": "Selenium2 (WebDriver) driver for Mink framework",
+    "keywords": [
+        "selenium",
+        "webdriver",
+        "javascript",
+        "ajax",
+        "testing",
+        "browser"
+    ],
+    "homepage": "http://mink.behat.org/",
+    "type": "mink-driver",
+    "license": "MIT",
     "authors": [
         {
-            "name":      "Pete Otaqui",
-            "email":     "pete@otaqui.com",
-            "homepage":  "https://github.com/pete-otaqui"
+            "name": "Pete Otaqui",
+            "email": "pete@otaqui.com",
+            "homepage": "https://github.com/pete-otaqui"
+        },
+        {
+            "name": "Konstantin Kudryashov",
+            "email": "ever.zet@gmail.com",
+            "homepage": "http://everzet.com"
+        }
+    ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:oleg-andreyev/driver-testsuite.git"
         },
         {
-            "name":      "Konstantin Kudryashov",
-            "email":     "ever.zet@gmail.com",
-            "homepage":  "http://everzet.com"
+            "type": "vcs",
+            "url": "git@github.com:oleg-andreyev/php-webdriver.git"
         }
     ],
-
     "require": {
-        "php":                       ">=5.3.1",
-        "behat/mink":                "~1.7@dev",
-        "instaclick/php-webdriver":  "~1.1"
+        "php": ">=5.3.1",
+        "ext-zip": "*",
+        "ext-curl": "*",
+        "ext-json": "*",
+        "behat/mink": "~1.7@dev",
+        "facebook/webdriver": "dev-adjust-getRelatedElements"
```

Comment:
You're not using a stable release. Why?

Building off an integration based on locally modified fork seems to be a bad idea.